include/nbl/video/SPhysicalDeviceFeatures.h

```diff
@@ -330,427 +330,1283 @@ struct SPhysicalDeviceFeatures
 
     /* Vulkan Extensions */
 
-    // [TODO] need impl or just expose `shader_tile_image` instead?
-    /* RasterizationOrderAttachmentAccessFeaturesARM *//* VK_ARM_rasterization_order_attachment_access */
-    bool rasterizationOrderColorAttachmentAccess = false;
-    bool rasterizationOrderDepthAttachmentAccess = false;
-    bool rasterizationOrderStencilAttachmentAccess = false;
-    
-    // [DO NOT EXPOSE] Vulkan 1.3 non-optional, we just enable them if available or else we need to make format support query functions in LogicalDevice as well
-    /* 4444FormatsFeaturesEXT *//* VK_EXT_4444_formats */
+    // [DO NOT EXPOSE] Instance extension & should enable implicitly if swapchain is enabled
+    /* VK_KHR_surface */
+
+    /* Extensions Exposed as Features: */
+
+    /* VK_KHR_swapchain */
+    /* Dependant on `IAPIConnection::SFeatures::swapchainMode` enabled on apiConnection Creation */
+    core::bitflag<E_SWAPCHAIN_MODE> swapchainMode = E_SWAPCHAIN_MODE::ESM_NONE;
+
+    // [DO NOT EXPOSE] We don't support yet
+    // VK_KHR_display
+
+    // [TODO] handle with a single num
+    /* VK_KHR_display_swapchain */
+
+    // [DO NOT EXPOSE] We don't support yet
+    // VK_KHR_xlib_surface
+
+    // [DO NOT EXPOSE] We don't support yet
+    // VK_KHR_xcb_surface
+
+    // [DO NOT EXPOSE] We don't support yet
+    // VK_KHR_wayland_surface
+
+    // [DO NOT EXPOSE] We don't support yet
+    // VK_KHR_mir_surface
+
+    // [DO NOT EXPOSE] We don't support yet
+    // VK_KHR_android_surface
+
+    // [DO NOT EXPOSE] We don't support yet
+    // VK_KHR_win32_surface
```

Comment:
put them all together with one comment `[DO NOT EXPOSE] OS-specific we enable implicitly as we detect the platform`